### PR TITLE
Show number of files/folders ignored, install attr as deps in Makefile

### PR DESCRIPTION
**Pull request by haoadoreorange**
Signed-off-by: HaotheAdorableOyster <github@comtam.dev>

### CODE REVIEW

**Comment by haoadoreorange** [comment]
No idea why it fails the test

File: Makefile [hunk 1]
```diff
@@ -7,11 +7,13 @@ test :
 
 # install dropboxignore
 install :
+	sudo apt install -y attr
```
Comment (sp1thas) [inline Makefile]: The purpose of dropbox ignore is be as cross-platform as possible. `apt` is the package manager for debian and debian-based systems. As a result, this `Makefile`could not be used by other platforms.
```suggestion
```

File: Makefile [hunk 2]
```diff
@@ -7,11 +7,13 @@ test :
...
 	cp bin/dropboxignore.sh ${DESTDIR}/bin/dropboxignore
 	chmod +x ${DESTDIR}/bin/dropboxignore
 	dropboxignore version
 
 # uninstall dropboxignore
 uninstall :
+	echo "\e Run `sudo apt purge -y attr` if no longer used"
```
Comment (sp1thas) [inline Makefile]: same here
```suggestion
```

File: bin/dropboxignore.sh [hunk 3]
```diff
@@ -313,14 +314,17 @@ ignore_file () {
   if [ "$(dirname "$1")" == "$DROPBOX_IGNORE_FILE_NAME" ]; then
     log_debug "Bypass $(realpath --relative-to="$BASE_FOLDER" "$1")"
   elif [  -z "$FILE_ATTR_VALUE" ]; then
+    if [ -f "$1" ]; then
```
Comment (sp1thas) [inline bin/dropboxignore.sh]: great fix

**Review by sp1thas — CHANGES_REQUESTED** [review]

**Comment by sp1thas** [comment]
Thanks @babyadoresorange for this PR, regarding the failed test, ignore it, will be fixed by #11

**Comment by haoadoreorange** [comment]
@sp1thas I also suggest to change the shebang to #!/bin/bash for security reason

**Comment by sp1thas** [comment]
@babyadoresorange sure, feel free to change the shebang

**Pull request closed without merging**